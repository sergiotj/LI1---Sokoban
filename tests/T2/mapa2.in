###################
#####   ###########
#####   ###########
#####   ###########
###      ##########
### # ## ##########
#   # ## #####  ..#
#               ..#
##### ### # ##  ..#
#####     #########
###################
11 2
5 8
7 7
5 6
7 6
16 3
17 3
