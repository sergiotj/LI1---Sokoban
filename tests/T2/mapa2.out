    #####
    #   #
    #H  #
  ###  H##
  #  H H #
### # ## #   ######
#   # ## #####  ..#
#               II#
##### ### #o##  ..#
    #     #########
    #######
